VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -172,7 +173,7 @@ public void onClick(View v) {
                 Activity act = getActivity();
                 ((BaseFragmentActivity) act).closeDrawer();
 
-                if (!(act instanceof MyVideosTabActivity)) {
+                if (!(act instanceof MyVideosActivity)) {
```

Comment:
any ESSY way to avoid using instanceof? Maybe an enum in the base fragment activity class representing the type of activity?